sys/include/net/gnrc/netif/6lo.h

```diff
@@ -18,12 +18,26 @@
 #ifndef NET_GNRC_NETIF_6LO_H
 #define NET_GNRC_NETIF_6LO_H
 
+#include <stdbool.h>
 #include <stdint.h>
 
 #ifdef __cplusplus
 extern "C" {
 #endif
 
+/**
+ * @name    Local 6LoWPAN capability flags
+ * @anchor  net_gnrc_netif_6lo_local_flags
+ * @see     gnrc_netif_6lo_t::local_flags
+ * @{
+ */
+/**
+ * @brief   Selective Fragment Recovery
+ * @see     [RFC 8931](https://tools.ietf.org/html/rfc8931)
+ */
+#define GNRC_NETIF_6LO_LOCAL_FLAGS_SFR  (0x01)
```

Comment:
Any interface capable of classic 6LoWPAN fragmentation should be capable of SFR. However, one might not activate this when e.g. other nodes in the network do not support SFR (it is not specified yet how this can be determined, but it most likely will use a flag 6LoWPAN capability option in the Router Advertisement).

Reply:
so maybe call this Selective Fragment Recovery *enabled* then, then it's clearer it's a setting and now a hardware capability

Reply:
None of these flags refer to capability, so I don't understand the confusion.

Reply:
> Local 6LoWPAN capability flags

Reply:
That comes from just reading Github and not your own doc ^^"

Reply:
Mhhh now that I read my doc again: I named them capability flags, since that is what [the associated option](https://tools.ietf.org/html/rfc7400#section-3.3) is called. I try to clarify however, that these flags are less about hardware restraints and more about the network status quo.

Reply:
I think changing the description to 

> Selective Fragment Recovery enabled

would already be enough to signal that this is a toggle

Reply:
Sorry only saw your comment now. I added a longer description on the nature of these flags but also will add the verb as proposed.